frida_mode/src/instrument/instrument_x64.c

```diff
@@ -326,7 +328,7 @@ void instrument_coverage_optimize_init(void) {
   FVERBOSE("Low address: %p", low_address);
 
   if (low_address == 0 ||
-      GPOINTER_TO_SIZE(low_address) > ((2UL << 20) - __afl_map_size)) {
+      GPOINTER_TO_SIZE(low_address) > ((2UL << 30) - __afl_map_size)) {
```

Comment:
We actually want our memory to be in the low 2GB of the address space. This was broken before.